spec/models/user_spec.rb

```diff
@@ -102,7 +102,7 @@
       let(:rolled_over_user) { build(:user, :rolled_over, associated_with_accredited_body: false, notifications_configured: true) }
 
       it "raises an error when trying to change state from 'rolled_over' to 'notifications_configured'" do
-        expect { rolled_over_user.accept_notifications_screen! }.to raise_error(AASM::InvalidTransition)
+        expect { rolled_over_user.accept_notifications_screen! }.to_not raise_error
```

Comment:
Asserting an error is not raised - test description needs to be updated

Reply:
Done!